Client/Frontend/Autofill/CreditCard/CreditCardSettingsViews+Model/CreditCardSettingsViewModel.swift

```diff
@@ -0,0 +1,50 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Shared
+import Storage
+import UIKit
+
+enum CreditCardSettingsState: String, Equatable, CaseIterable {
```

Comment:
The state is not used in this model at all - I think this should be move to another class.